cfme/base/login.py

```diff
@@ -43,7 +43,10 @@ def current_username(self):
 
     @property
     def current_fullname(self):
-        return self.settings.text.strip().split('|', 1)[0].strip()
+        try:
+            return self.settings.text.strip().split('|', 1)[0].strip()
+        except AttributeError:
```

Comment:
Is there anything that could be said in a comment about why we would get that attribute error, or the named exception enough info?   My thinking is you saw this error under a specific condition and hence commenting on that condition may be helpful for future you or me.

Reply:
As in the methods above, it ends up happening because there's no check that the view is actually displayed.  An AttributeError is raised because `self.settings.text` is None, which has no `strip()` attr. Will put in comment.